superset/connectors/sqla/models.py

```diff
@@ -1218,7 +1218,10 @@ def fetch_metadata(self, commit: bool = True) -> None:
 
     @classmethod
     def import_obj(
-        cls, i_datasource: "SqlaTable", import_time: Optional[int] = None
+        cls,
+        i_datasource: "SqlaTable",
```

Comment:
Not yours, but the name `i_datasource` makes me uncomfortable 😬

Reply:
"input" ?

Reply:
Let's leave that to another PR